virttest/libvirt_version.py

```diff
@@ -24,16 +25,22 @@ def version_compare(major, minor, update):
     :param major: Major version to compare against
     :param minor: Minor version to compare against
     :param update: Update value to compare against
+    :param session: Shell session on remote host
     :return: True if running version is greater than or
                   equal to the input libvirt version
     """
     global LIBVIRT_LIB_VERSION
 
+    func = process.system_output
+    if session:
+        func = session.cmd_output
+        LIBVIRT_LIB_VERSION_ORG, LIBVIRT_LIB_VERSION = LIBVIRT_LIB_VERSION, 0
+
     if LIBVIRT_LIB_VERSION == 0:
         try:
             regex = r'[Uu]sing\s*[Ll]ibrary:\s*[Ll]ibvirt\s*'
             regex += r'(\d+)\.(\d+)\.(\d+)'
-            lines = process.run("virsh version").stdout_text.splitlines()
+            lines = decode_to_text(func("virsh version")).splitlines()
```

Comment:
Seems 'session.cmd_output' does not need to use decode_to_text?

Reply:
No, have to. It was bytes without decode_to_text.